esp/esp/program/modules/handlers/programprintables.py

```diff
@@ -934,26 +1074,35 @@ def getSchedule(program, user, schedule_type=None, room_numbers=True, include_da
             elif user.isVolunteer():
                 schedule_type = u'Volunteer'
 
-
+        include_roles = None
+        pretty_schedule_type = schedule_type
         if schedule_type == u'Student':
             template = get_template('program/modules/programprintables/studentschedule_email.html')
             sched_items = ProgramPrintables.get_student_classlist(program, user)
-            sched_items.sort()
         elif schedule_type == u'Teacher':
             template = get_template('program/modules/programprintables/teacherschedule_email.html')
-            sched_items = ProgramPrintables.get_teacher_classlist(program, user)
-            sched_items.sort()
+            sched_items = ProgramPrintables.get_teacher_classlist(program, user, teaching = True, moderating = False)
+        elif schedule_type == u'TeacherModerator':
+            include_roles = True
+            pretty_schedule_type = u'Teacher and ' + program.getModeratorTitle().lower()
```

Comment:
```suggestion
            pretty_schedule_type = u'Teacher and ' + program.getModeratorTitle()
```

Reply:
I think the word "Schedule" with caps gets put after this, so I followed titlecase, but i can confirm that